### PR TITLE
feat(map-marker): add mouse events

### PR DESCRIPTION
**Pull request by NathanDeVuono**
[Jira](https://urbandoor.atlassian.net/browse/UD-6118)

### CODE REVIEW

File: src/components/press-map-marker/press-map-marker.vue [hunk 1]
```diff
@@ -3,6 +3,8 @@
 </template>
 
 <script>
+const listenerTypes = ['mouseover', 'mouseout'];
```
Comment (bzang) [inline src/components/press-map-marker/press-map-marker.vue]: Thoughts on making this uppercase for "global" consts? i.e. `LISTENER_TYPES`

Comment (NathanDeVuono) [reply src/components/press-map-marker/press-map-marker.vue]: I'm indifferent, they're component local constants and not really useful outside the file.

I'd personally prefer to bind the listeners individually as to be more explicit - the execution cost is the same.

Comment (NathanDeVuono) [reply src/components/press-map-marker/press-map-marker.vue]: Looks like two votes for, I'll make the change

Comment (bzang) [reply src/components/press-map-marker/press-map-marker.vue]: Mostly just a visual cue

**Pull request merged**